ui/src/components/NewExperimentNext/data/types.ts

```diff
@@ -144,6 +144,30 @@ const ioCommon: Spec = {
   },
 }
 
+const jvmClassAndMethod: Spec = {
+  class: {
+    field: 'text',
+    label: 'Class',
+    value: '',
+    helperText: 'Class that is injected with faults',
+  },
+  method: {
+    field: 'text',
+    label: 'Method',
+    value: '',
+    helperText: 'Method that is injected with faults',
+  },
+}
+
+const jvmPort: Spec = {
+  port: {
+    field: 'number',
+    label: 'Port',
+    value: 9288,
```

Comment:
I prefer to not set this default value, or keep consistent with `DefaultJVMAgentPort`, `9277`.

rest LGTM!

Reply:
PTAL @WangXiangUSTC

Reply:
I update it to 9277

Reply:
I don't think users should care about this configuration unless it uses the same port number, so give it a default value